macstadium-staging/workers.tf

```diff
@@ -0,0 +1,142 @@
+variable "latest_travis_worker_version" {}
```

Comment:
I think we should decide how we want the worker version to behave in staging.

In the existing configuration, the `travis_worker_version` variable was used for non-staging workers, and `latest_travis_worker_version` was used for staging workers. The latter variable is populated automatically by a script using the most-recent GitHub release of worker.

In my experience, while that may be nice for making sure staging is always running a recent worker, it's not really what I wanted and it made things clumsy when I wanted to test a new worker version on staging. I think I would prefer that we remove `latest_travis_worker_version`, and use the explicit `travis_worker_version` here for the staging workers, similar to how production works. This is even more feasible now, since the variable will be distinct for the two environments, so we can easily test a new version in staging.